lib/figleaf/settings.rb

```diff
@@ -84,7 +84,7 @@ def load_rb_file(file_path, env = nil)
       end
 
       def load_yaml_file(file_path)
-        YAML.load(ERB.new(IO.read(file_path)).result)
+        YAML.unsafe_load(ERB.new(IO.read(file_path)).result)
```

Comment:
I doubt we're relying on anything outside the allowlist (`TrueClass, FalseClass, NilClass, Integer, Float, String, Array, Hash`), but I understand if you don't want to change this for backwards compatibility
```suggestion
        YAML.safe_load(ERB.new(IO.read(file_path)).result)
```